docs/includes/apiargs-MongoDBDatabase-method-createCollection-option.yaml

```diff
@@ -133,6 +133,19 @@ interface: phpmethod
 operation: ~
 optional: true
 ---
+arg_name: option
+name: timeseries
```

Comment:
I realize this PR pertains to [DRIVERS-1438](https://jira.mongodb.org/browse/DRIVERS-1438), which only added a `timeseries` option, but there is a follow-up ticket ([DRIVERS-1774](https://jira.mongodb.org/browse/DRIVERS-1774)) that moves `expireAfterSeconds` to a top-level option. That spec PR is still being reviewed, so there are not yet any language tickets.

I'd suggest holding off on this PR for the time being, since DRIVERS-1774 will require changing this documentation, `CreateCollection.php`, and resyncing spec tests.

Reply:
Ah, thanks for the heads up. I'll give the tests from that PR a quick check to provide feedback and will wait until DRIVERS-1774 is ready before merging.